dist/backdraft.js

```diff
@@ -845,11 +845,19 @@ _.extend(Plugin.factory, {
     ',
     sFilterListTemplate : '\
       <li><label><input class="list" id="value" type="checkbox" name="{{= attr }}" \
-         value="{{= options }}" /> <%= options %></label></li> \
+         value="{{= options }}" /> {{= options }}</label></li> \
+    ',
+    sActionButtonTemplate : '\
+      <button class="btn btn-primary btn-sm {{= klass}}" name="button" type="submit" title="">{{= caption}}</button>\
```

Comment:
called it klass because class causes compiling issues.  if you have a better idea for the name let me know.
also, ActionButton isn't the best name for this control.  any other thoughts on that would be appreciated.